doc/060_forget.rst

```diff
@@ -82,21 +75,42 @@ command must be run:
 
     $ restic -r /srv/restic-repo prune
     enter password for repository:
-
+    repository 33002c5e opened successfully, password is correct
+    get all snapshots
+    load indexes
+    find data that is still in use for 4 snapshots
+    [0:00] 100.00%  4 / 4 snapshots
+    find packs in index and calculate used size...
+    collect packs for deletion and repacking...
+    [0:00] 100.00%  9 / 9 packs processed
+    
+    used:                49 blobs / 10.654 MiB
+    duplicates:           0 blobs / 0 B
+    unused:              81 blobs / 18.355 MiB
+    unreferenced:                   0 B
+    total:              130 blobs / 29.010 MiB
+    unused size: 63.27% of total size
+    
+    to repack:          102 blobs / 28.936 MiB
+      -> prunes:         67 blobs / 18.302 MiB
+    to delete:           14 blobs / 54.268 KiB
+    delete unreferenced:            0 B
+    total prune:         81 blobs / 18.355 MiB
+    unused size after prune: 0.00% of total size
```

Comment:
Should this and possibly some other parts only be shown when having increased verbosity?

Reply:
I agree, if possible I’d keep the default and `-v` output unchanged (including the short “this frees this N Mb/Gb”) and add the extra information to `-vv`.

Reply:
I reduced the output a lot for low verbosity. Now just 5 lines.